robocop/checkers/duplications.py

```diff
@@ -76,6 +104,14 @@ def configure_sections_order(value):
         msg="Multiple library imports with name '{{ name }}' and identical arguments (first occurrence in line "
         "{{ first_occurrence_line }})",
         severity=RuleSeverity.WARNING,
+        docs="""
+        If you need to reimport library use alias::
+        
+            *** Settings ***
+            Library  RobotLibrary
+            Library  RobotLibrary  AS  OtherRobotLibrary
```

Comment:
I think it would be best to not use this example at all - duplicated library imports is quite straightforward so we can ignore docs for it.

Reply:
I partly agree but on the other hand we need to decide if we want to have an example for every rule or not. I think it might be confusing if some rules will have examples and some not so IMO it's better to have it for all rules (even if some of them are trivial).